### PR TITLE
Fix validation workflow for test channel

### PR DESCRIPTION
**Pull request by atalman**
Fix validation workflow for test chan, fix conda installs for 3.11

### CODE REVIEW

File: .github/workflows/validate-nightly-binaries.yml [hunk 1]
```diff
@@ -23,5 +23,5 @@ jobs:
   nightly:
     uses: ./.github/workflows/validate-binaries.yml
     with:
-      channel: nightly
```
Comment (atalman) [inline .github/workflows/validate-nightly-binaries.yml:26]: Please note will revert this change before pushing. This is to test, test chan

**Review by weiwangmeta — APPROVED** [review]
Please leave a note on why -c malfet is needed and merge when green. Looks like we have some conda failures.

**Review by osalpekar — APPROVED** [review]
